maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/utils/WiremockStubs.java

```diff
@@ -143,4 +143,11 @@ public static void stubForGetRepOrders(String response) {
                         .withHeader("Content-Type", String.valueOf(MediaType.APPLICATION_JSON))
                         .withBody(response)));
     }
+
+    public static void stubForUpdateSendToCCLF(String response) {
+        stubFor(put(urlMatching(MAAT_API_ASSESSMENT_URL + "/rep-orders/" + TestModelDataBuilder.REP_ID + "/send-to-cclf"))
+                .willReturn(WireMock.ok()
+                        .withHeader("Content-Type", String.valueOf(MediaType.APPLICATION_JSON))
+                        .withBody(response)));
```

Comment:
I don't think we actually care about the response, I'd probably just update the `APIService` to return void

Reply:
Updated